scv-prototype-frontend/src/app/App.jsx

```diff
@@ -35,13 +35,13 @@ const App = () => {
           <PageMetadataProvider>
             <BrowserRouter>
               <Switch>
-                <Route path="/" exact>
-                  <Redirect to="/private" />
+                <Route path='/' exact>
+                  <Redirect to='/private' />
                 </Route>
-                <PrivateRoute path="/private" component={Private} />
-                <Route path="/msca" component={MSCA} exact />
-                <Route path="/sign-in" component={Login} exact />
-                <Route path="*" component={Error404} />
+                <PrivateRoute path='/private' component={Private} />
+                <Route path='/msca' component={MSCA} exact />
+                {/* <Route path="/sign-in" component={Login} exact /> */}
+                <Route path='*' component={Error404} />
```

Comment:
JSX attributes should be quoted with double quotes, not single quotes.  
See https://github.com/airbnb/javascript/tree/master/react#quotes